app-backend/batch/src/main/scala/geojsonImport/ImportGeojsonFiles.scala

```diff
@@ -0,0 +1,170 @@
+package com.rasterfoundry.batch.geojsonImport
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.ExecutionContext
+import com.rasterfoundry.batch.util.conf.Config
+import com.rasterfoundry.database.util.RFTransactor
+import com.rasterfoundry.datamodel._
+import com.rasterfoundry.common.S3
+
+import com.typesafe.scalalogging.LazyLogging
+
+import doobie.ConnectionIO
+import doobie.implicits._
+import cats.implicits._
+import cats.effect._
+
+import scala.util._
+import java.util.UUID
+import java.util.concurrent.Executors
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder
+
+import java.util.UUID
+import com.rasterfoundry.database.GeojsonUploadDao
+import com.amazonaws.services.s3.AmazonS3URI
+import java.net.URLDecoder
+
+import java.nio.charset.StandardCharsets
+import io.circe.parser.decode
+import com.rasterfoundry.database.UserDao
+import com.rasterfoundry.database.AnnotationDao
+
+object CommandLine {
+  final case class Params(
+      upload: UUID = UUID.randomUUID(),
+      testRun: Boolean = false
+  )
+
+  val parser =
+    new scopt.OptionParser[Params]("raster-foundry-geojson-import") {
+      override def terminate(exitState: Either[String, Unit]): Unit = ()
+
+      head("raster-foundry-geojson-import", "0.1")
+
+      opt[Unit]('t', "test")
+        .action(
+          (_, conf) => conf.copy(testRun = true)
+        )
+        .text("Dry run geojson import to verify geojson file parses correctly to annotations")
+
+      opt[String]('u', "upload")
+        .required()
+        .action(
+          (u, conf) => {
+            conf.copy(upload = UUID.fromString(u))
+          }
+        )
+        .text("GeojsonUpload entry to process")
+    }
+}
+
+object ImportGeojsonFiles extends Config with LazyLogging {
```

Comment:
Since this is a job... it should probably extend `Job`. That will also save you having to create a context shift.